src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/OpenTelemetryProtocolExporterEventSource.cs

```diff
@@ -103,5 +103,11 @@ public void MissingPermissionsToReadEnvironmentVariable(string exception)
         {
             this.WriteEvent(7, exception);
         }
+
+        [Event(8, Message = "Unsupported protocol '{0}' is configured, default protocol will be used.", Level = EventLevel.Warning)]
+        public void UnsupportedProtocol(string protocol)
```

Comment:
nit: might be good to indicate which is the default here.

Reply:
Added default to message.